src/scripts/main.js

```diff
@@ -1,8 +1,26 @@
 'use strict';
 
-const people = require('./lib/people');
+const people = require('./lib/people.json');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
 
-// write your code here
+people.forEach(human => {
+  const { sex, born, died } = human;
+  const gender = (personSex) => personSex === 'm' ? 'Male' : 'Famele';
+  const nameHuman = human.name;
+  const age = human.died - human.born;
+  const century = Math.ceil(human.died / 100);
+  const arr = [nameHuman, gender(sex), born, died, age, century];
+  const row = document.createElement('tr');
+
+  for (let i = 0; i < arr.length; i++) {
+    const cell = document.createElement('td');
+
+    cell.textContent = `${arr[i]}`;
+    row.appendChild(cell);
+  }
```

Comment:
I'd recommend to use `row.innerHtml` instead of `loop`

Reply:
still not fixed, you can put all html structure using `innerHtml` 
[documentation](https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML)